api/models/Product/ProductType.js

```diff
@@ -0,0 +1,31 @@
+const {
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLString,
+} = require('graphql');
+
+
+const ProductType = new GraphQLObjectType({
+  name: 'Product',
+  description: 'TThis represents to product inventory for the restaurant',
+  fields: () => ({
+    id: {
+      type: GraphQLInt,
+      resolve: (product) => product.id,
+    },
+    name: {
+      type: GraphQLString,
+      resolve: (product) => product.name,
+    },
+    description: {
+      type: GraphQLString,
+      resolve: (product) => product.description,
+    },
+    price: {
+      type: GraphQLString,
```

Comment:
As said before, this should be a number, depending on what type you are using you should check for GraphQL type and use the same